jekyll/_includes/snippets/features-of-circleci.adoc

```diff
@@ -0,0 +1,24 @@
+=== Private orbs
+CircleCI orbs are shareable configuration packages that enable developers to create private, standardized configurations for use across an organization by different teams.
+
+For more information on private orbs, refer to the <<orb-intro/#private-orbs-vs-public-orbs,Orbs Introduction>> page.
+
+=== Concurrency and test splitting
+The ability to run multiple tests at the same time (concurrently) is a powerful feature that allows you to decrease your build times and shorten feedback cycles by running jobs concurrently with workflows.
+
+For more information about how to utilize test splitting, refer to the <<parallelism-faster-jobs#,Running Tests In Parallel>> page.
```

Comment:
There's a slight issue with language here. In CCI concurrency and parallelism have specific meanings. Test splitting - running tests across multiple containers in in parallel is parallelism. Concurrency refers to running _jobs_ concurrently through the use of workflows. So in these plan docs where we talk about customisable concurrency, we need to change that to parallelism.

Max concurrency (the max number of jobs a customer can run at the same time) is not a customer-facing setting. Unless they have surfaced that maybe?

Test splitting defo is parallelism though.

I know the terms can be seen as interchangeable but this was decided upon to avoid confusion between the two concepts between the product

Reply:
@rosieyohannan I can take a look, but this needs to mirror the pricing page and the pricing page says Concurrency. Maybe there was miscommunication when this was originally written (this is from the Free plan information that exists now in the Docs that Thomas worked on and was approved by marketing). Maybe we just need to leave the test-splitting and parallelism out of this completely.